bin/bout-v5-input-file-upgrader.py

```diff
@@ -9,13 +9,16 @@
 from boutdata.data import BoutOptionsFile
 from boututils.boutwarnings import AlwaysWarning
 
-# This should be a list of dicts, each containing exactly two keys:
-# "old" and "new". The values of these keys should be the old/new
-# names of input file values or sections
+# This should be a list of dicts, each containing "old", "new" and optionally "values".
+# The values of "old"/"new" keys should be the old/new names of input file values or
+# sections. The value of "values" is a dict containing replacements for values of the
+# option.
 REPLACEMENTS = [
     {"old": "mesh:paralleltransform", "new": "mesh:paralleltransform:type"},
     {"old": "fci", "new": "mesh:paralleltransform"},
     {"old": "interpolation", "new": "mesh:paralleltransform:xzinterpolation"},
+    {"old": "fft:fft_measure", "new": "fft:fft_measurement_flag",
+        "values": {"false": "estimate", "true": "measure"}}
```

Comment:
BOUT++ actually converts `y/Y/t/T/1` to `true` and `n/N/f/F/0` to `false`. I'm not sure what the best way to handle that here is, without this suddenly becoming massively complicated. My first thought was to add another key, `type`, and if that's `bool`, apply a conversion function first. Then use `True`/`False` literals in the `value` dict.

Reply:
Agreed. I've made an update implementing a `parse_bool()` function. I think it's not too horrible.

I've tested `bout-v5-input-file-upgrader.py` by 'upgrading' `fft_measure` from all of `true/y/Y/t/T/1` and `false/n/N/f/F/0`.

Reply:
Lovely, thanks @johnomotani ! We can now deprecate both input keys and values with a nice upgrade path :tada: